includes/wikia/DefaultSettings.php

```diff
@@ -1797,3 +1798,8 @@
  * flag for using (possibly external) service for managing a user's preferences
  */
 $wgPreferencesUseService = false;
+
+/**
+ * Parser Tag
+ */
+require_once "$IP/extensions/wikia/SoundCloudTag/SoundCloudTag.setup.php" ;
```

Comment:
As far as I remember, we can put site-wide enabled extension in this file. See https://github.com/Wikia/app/pull/8406#discussion-diff-39293694R606

Can you please share an idea about better place to put those?